update.go

```diff
@@ -204,6 +202,28 @@ other options are ignored.
 					*pair.dest = uint64(v)
 				}
 			}
+
+			for _, pair := range []struct {
+				opt  string
+				dest *uint64
+			}{
+				{"memory", r.Memory.Limit},
+				{"memory-swap", r.Memory.Swap},
```

Comment:
The rule that '-1' means reset applies to all `*.limit_in_bytes`, so you can do this for all memory related limitations.